conf/app.ini

```diff
@@ -126,6 +126,8 @@ SSH_KEY_TEST_PATH =
 SSH_KEYGEN_PATH = ssh-keygen
 ; Enable SSH Authorized Key Backup when rewriting all keys, default is true
 SSH_BACKUP_AUTHORIZED_KEYS = true
+; Enable exposure of SSH clone URL to anonymous visitors, default is true
+SSH_EXPOSE_ANONYMOUS = true
```

Comment:
I think default should be false to match GitHub and also for security reason not to expose unneeded URL's